platforms/documentation/docs/src/snippets/bestPractices/avoidInternal/groovy/build.gradle

```diff
@@ -0,0 +1,21 @@
+// tag::avoid-this[]
+abstract class BadTask extends org.gradle.api.internal.AbstractTask { // <1>
```

Comment:
💅 Did you mean to leave the `// <1>` callouts in these examples?  I think it might be helpful to use them to emphasize that AbstractTask is internal, as the simple name alone seems correct.  Each number should only be present once though - I'm kind of surprised this isn't a build failure.

Reply:
It was intentional, I updated to use unique numbers. Is there a way to highlight just a part of the line?